app/serializers/wizard_field_serializer.rb

```diff
@@ -75,6 +75,14 @@ def include_icon?
     object.icon.present?
   end
 
+  def disabled
+    object.disabled
+  end
+
+  def include_disabled?
+    object.disabled.present?
```

Comment:
```suggestion
    object.disabled
```

We don't need to include "disabled: false" attrs

Reply:
Made the changes, thanks for the suggestion! 👍🏼